app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/projectManager.ts

```diff
@@ -0,0 +1,239 @@
+/** @file This module defines the Project Manager endpoint. */
+import * as newtype from '../newtype'
+
+// =================
+// === Constants ===
+// =================
+
+/** The address of the socket for the project manager. */
+const PROJECT_MANAGER_ENDPOINT = 'ws://127.0.0.1:30535'
```

Comment:
Is this a hardcoded thing? Is this agreed with backend? If so, docs should mention which other part of the project it should be synchronized with.

Reply:
should the other side also have a comment pointing to this file?

Reply:
CC @PabloBuchu - I was chatting with Pawel how to refactor it to a common file.

Reply:
It should be placed in `app/gui/config.yaml` so both rust and js can access it

Reply:
should this be done in this PR?

Reply:
from our side imho yes. Let's put the `const PROJECT_MANAGER_ENDPOINT = 'ws://127.0.0.1:30535'` to `gui/config.yaml` and just use it in `projectManager.ts`
```
import GLOBAL_CONFIG from '../../../../gui/config.yaml' assert { type: 'yaml' }
...
return (this.instance ??= new ProjectManager(GLOBAL_CONFIG.projectManagerEndpoint))
```